src/sentry/static/sentry/app/components/modals/addDashboardWidgetModal.tsx

```diff
@@ -24,15 +24,23 @@ import {generateFieldOptions} from 'app/views/eventsV2/utils';
 import SelectField from 'app/views/settings/components/forms/selectField';
 import TextField from 'app/views/settings/components/forms/textField';
 
-type Props = ModalRenderProps & {
-  api: Client;
+// forwarded from src/sentry/static/sentry/app/actionCreators/modal.tsx
```

Comment:
Should we import/export the DashboardWidgetModalOptions and use `Pick` to not duplicate some of the properties?

Reply:
Would we still have code splitting if we import `DashboardWidgetModalOptions`?

Reply:
I don't think types should impact code splitting as the output of typescript shouldn't leave imports behind.